### PR TITLE
Standalone executable

### PR DESCRIPTION
**Pull request by mbland**
Needs better input parsing and documentation, but is a functional proof-of-concept. Build an identical index to the original plugin and copied over the bundle files using:

``` shell
$ mkdir -p _tmp/_site/api/v1
$ cp ../hub/_site/api/v1/pages.json _tmp/_site/api/v1/
$ bundle exec bin/jekyll_pages_api_search _tmp/_site ../hub/_config.yml \
  _tmp/_site/api/v1/pages.json
```

Could stand to provide options to output the Liquid tags as Hugo shortcodes. Still, a very promising initial proof-of-concept.

cc: @afeld

### CODE REVIEW

File: bin/jekyll_pages_api_search [hunk 1]
```diff
@@ -0,0 +1,66 @@
+#! /usr/bin/env ruby
+# Author: Mike Bland <michael.bland@gsa.gov>
+# Date:   2015-06-21
+
+# Monkeypatch to avoid "undefine method `render` error.
```
Comment (afeld) [inline bin/jekyll_pages_api_search]: Any idea what is causing that?

**Comment by afeld** [comment]
Good start!

/cc https://github.com/18F/jekyll_pages_api/pull/27

File: bin/jekyll_pages_api_search (hunk 1 above)
Comment (mbland) [inline bin/jekyll_pages_api_search]: So in 18F/jekyll_pages_api#27 I've updated `lib/jekyll_pages_api.rb` to explicitly include all of its components, and the problem appears to go away when I modify my Gemfile to use that updated version.

**Comment by mbland** [comment]
@afeld This is ready for another look! :-)

File: bin/jekyll_pages_api_search [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+output[pages_json] = site.pages.first.output unless File.exist? pages_json
+output.each do |outfile, content|
+  File.open(outfile, 'w') {|f| f << content}
+  JPAS::Compressor::gzip_in_memory_content outfile => content
+end
+
+# Copy the search bundle files.
+target_path = File.join site.source, ASSETS_DIR
+FileUtils.mkdir_p target_path
+JPAS::JavascriptCopier::search_bundle_paths {|f| FileUtils.cp f, target_path}
```
Comment (afeld) [inline bin/jekyll_pages_api_search]: You might want to move this logic out to `lib/` and add some tests. Feel free to do in a separate PR if you like.

**Comment by afeld** [comment]
Looks good! Just the one comment, though feel free to merge this and do it separately.

**Comment by mbland** [comment]
Went ahead and moved the code and added tests. @afeld: PTAL?

**Comment by afeld** [comment]
Tests failed :cry:

**Comment by mbland** [comment]
Ah, that's a Rakefile config problem. Should be fixed now. Also added logic to address the empty bundle problem from #6 and #7.

**Pull request merged**